client/me/help/help-contact/index.jsx

```diff
@@ -204,6 +207,26 @@ module.exports = React.createClass( {
 
 	onOperatorsAway: function() {
 		this.showOperatorAvailabilityNotice( false );
+
+		//Autofill the subject field since we will be showing it now that operators have went away.
+		this.autofillSubject();
+	},
+
+	/**
+	 * Auto fill the subject with the first five words contained in the message field of the contact form.
+	 */
+	autofillSubject: function() {
+		const contactForm = this.state.contactForm;
+
+		if ( ! contactForm.message || contactForm.subject ) {
+			return;
+		}
+
+		const words = contactForm.message.split( /\s+/ );
+
+		contactForm.subject = words.slice( 0, 5 ).join( ' ' ) + '…';
```

Comment:
React strongly advices against mutating the component state outside of `setState()`, which is what happens here when we set `contactForm.subject` to the new string. This would be a good place to use `Object.assign()` or the destructuring assignment if it's available to us…

```js
this.setState( { ...contactForm, subject: words.slice( 0, 5 ).join( ' ' )  + '-' } );
```